src/xmlsec/__init__.py

```diff
@@ -6,9 +6,12 @@
 __author__ = 'leifj'
 
 import six
+import logging
+# Instantiate log here to avoid circular dependency
+log = logging.getLogger('xmlsec')
+
```

Comment:
This whole file does not need to change. If we are to change something it should be the logger name.

```python
log = logging.getLogger(__name__)
```